models/iaf_psc_delta.h

```diff
@@ -57,45 +125,25 @@ spikes are added to the membrane potential at the end of the
 refractory period, dampened according to the interval between
 arrival and end of refractoriness.
```

Comment:
I think the first sentence ("The threshold crossing ...") can also be remove, that is discussed above.

The second part ("Spikes arriving ...") might go into a Note, since it is a bit unusual. The "set to true"  should be "set to ``True``", but I cannot make suggestions for that line here in Github.